exercises/dominoes/dominoes_test.rb

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'dominoes'
+
+# Test data version:
+# 82eb00d
```

Comment:
Ah, okay. One line it is. I think putting it on a separate line was an artifact of the time when both the sha and the git commit summary was included... nowadays since it's just the sha it probably makes sense on one line.